docs/migrating.md

```diff
@@ -1 +1,84 @@
+| `SIGNALFX_PROFILER_PROCESSES`         | Sets the filename of executables the profiler can attach to. Supports multiple semicolon-separated values, for example: `MyApp.exe;dotnet.exe`       |                                                   | none                                                                                                       |
+| `SIGNALFX_TRACE_CONFIG_FILE`          | Path of the JSON configuration file.                                                                                                                 |                                                   | none                                                                                                       |
+| `SIGNALFX_TRACE_ENABLED`              | Enable to activate the tracer.                                                                                                                       | `true`                                            | none                                                                                                       |
+| `SIGNALFX_METRICS_{0}_ENABLED`        | Configuration pattern for enabling or disabling a specific group of metrics.                                                                         | `false`                                           | `OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS`/ `OTEL_DOTNET_AUTO_METRICS_DISABLED_INSTRUMENTATIONS`  |
+
+## Global instrumentation settings
+
+| Setting                              | Description                                                                                                                                                                                                                                | Default   | OpenTelemetry equivalent                                                                                              |
+|--------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|-----------------------------------------------------------------------------------------------------------------------|
+| `SIGNALFX_DISABLED_INTEGRATIONS`     | Comma-separated list of disabled library instrumentations.                                                                                                                                                                                 |           | `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS`                                                                   |
```

Comment:
I think that information about supported integrations has to be included. The value is not 1:1 mapping.

Reply:
Or at least mention where it could be found.